tailwind-showcases/doc/tailwind-showcases.md

```diff
@@ -1,82 +1,160 @@
-import '../dockit-tailwind-showcases.define.js';
+```js script
+import '@divriots/dockit-core/tailwind-showcases/dockit-tailwind-showcases.define.js';
+```
+
+# dockit-tailwind-showcases
+
+Component to visualize Tailwind design tokens.
+
+## Setup
+
+```js
+import '@divriots/dockit-core/tailwind-showcases/dockit-tailwind-showcases.define.js';
+```
+
+## Demos
+
+We need the following imports for the demos below:
+
+```js preview-story
+import { html } from 'lit';
 import 'twind/shim';
 import twTheme from 'tailwindcss/defaultTheme.js';
-import { html } from 'lit';
+export const demoImports = () => `<p>Check the imports code 👇</p>`;
+```
 
+### Background-color
+
+```js preview-story
 export const bg_color = () => html`<dockit-tailwind-showcases
+  .theme=${twTheme}
   showcase-key="backgroundColor"
   component-class="shadow-xl h-24 w-24"
+  checkered-background
```

Comment:
due to a bug I fixed earlier this component now requires `checkered-background` which propapgates through all of the component to the underlying `dockit-box`

given it's quite weird and most of the time you need `checkered-background` anyway, here and in all other example we have in docs, I don't get why it's not a default and why we need this as an option instead of just always having `checkered-background`

time to rethink this part IMO

Reply:
Just remove checked-background and make it default always.
If someone doesn't want it it can implement his own box - it's not rocker science.

Reply:
fixed in https://github.com/divriots/dockit-core/pull/55